analyzer/src/funTest/assets/projects/synthetic/gomod/go.sum

```diff
@@ -1,10 +1,24 @@
 github.com/fatih/color v1.7.0 h1:DkWD4oS2D8LGGgTQ6IvwJJXSL5Vp2ffcQg58nFV38Ys=
 github.com/fatih/color v1.7.0/go.mod h1:Zm6kSWBoL9eyXnKyktHP6abPY2pDugNf5KwzbycvMj4=
+github.com/fatih/color v1.13.0 h1:8LOYc1KYPPmyKMuN8QV2DNRWNbLo6LZ0iLs8+mlH53w=
```

Comment:
Can the checksums of the old versions be removed?

Reply:
I've spent quite a while researching this but didn't find an answer which satisfied me.
I would prefer to investigate that later on. Are you fine with that?

Note: I've experimented removing the obsolete versions manually. If you then run `go mod tidy` after manual removal it would re-add the removed entries again. I haven't figured out exactly why this is so. But I've read that old versions are kept in the sum file for future downgrades of dependencies. So, it's ensured that an upgrade followed by a downgrade yields using the dependency with the same hash. So, it's a security feature.

Reply:
Thanks for the research.

Reply:
I've had it done already before you asked this question. Anyhow, NP. ;-)